Podfile.lock

```diff
@@ -139,6 +140,7 @@ DEPENDENCIES:
   - GoogleMaps (= 4.1.0)
   - GooglePlaces (= 4.1.0)
   - GzipSwift
+  - SwiftLint
```

Comment:
the community is slowly moving away from Pods, is it possible to add SwiftLint using SPM?

Reply:
I checked and for my knowledge it is not possible, at least for now